nx-dev/nx-dev/pages/_app.tsx

```diff
@@ -8,6 +8,35 @@ export default function CustomApp({ Component, pageProps }: AppProps) {
     <>
       <Head>
         <title>Nx: Smart, Extensible Build Framework</title>
+        <meta charSet="utf-8" />
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:site" content="@nxdevtools" />
+        <meta name="twitter:creator" content="@nrwl_io" />
+        <meta
+          name="twitter:title"
+          content="Nx: Smart, Extensible Build Framework"
+        />
+        <meta
+          name="twitter:description"
+          content="With Nx, you can develop multiple full-stack applications holistically and share code between them all in the same workspace. Add Cypress, Jest, Prettier, and Nest into your dev workflow."
+        />
+        <meta name="twitter:image" content="/images/nx-media.jpg" />
+        <meta
+          name="twitter:image:alt"
+          content="Nx: Smart, Extensible Build Framework"
+        />
+        <meta property="og:url" content="https://nx.dev" />
+        <meta
+          property="og:description"
+          content="With Nx, you can develop multiple full-stack applications holistically and share code between them all in the same workspace. Add Cypress, Jest, Prettier, and Nest into your dev workflow."
```

Comment:
Can i ask why Nest gets mentioned? I understand that the others get mentioned as dev tools but why nest? It is a framework for server side apps or am i wrong?

Reply:
That's a good point, do we want to replace it with Storybook?

Reply:
Hi @Phillip9587! Yes you are correct, Nest is a framework for building server applications and we do support and have generators for it. Though not sure we would want to name it specifically. We can add others, like Storybook or like Next.js, that would be a better fit.